build.py

```diff
@@ -2553,6 +2541,54 @@ def enable_all():
         required=False,
         help="Override specified backend CMake argument in the build as <backend>:<name>=<value>. The argument is passed to CMake as -D<name>=<value>. This flag only impacts CMake arguments that are used by build.py. To unconditionally add a CMake argument to the backend build use --extra-backend-cmake-arg.",
     )
+    parser.add_argument(
+        "--release-version",
+        required=False,
+        default=DEFAULT_TRITON_VERSION_MAP["release_version"],
+        help="Provide any release version.",
+    )
+    parser.add_argument(
+        "--triton-container-version",
+        required=False,
+        default=DEFAULT_TRITON_VERSION_MAP["triton_container_version"],
+        help="Provide any released version of project.",
```

Comment:
Is the `help` message make sense, I don't think we release any `dev` versions

Reply:
This text was previously present. I've only moved it to organize the args better. Happy to remove it.

Reply:
Can you suggest a better help message

Reply:
I think this parameter indicates current development version of the container, not necessary a released version. Based on this, in my vision, the parameter should be something like `triton-development-version`, and help message would be `Provide the current development version of the container`. I'm not sure how relevant it would be to provide the past development versions and how much confusion this would introduce to users. Do we really need to expose this parameter? @nv-kmcgill53 and @mc-nv , what do you think?

Reply:
Any kind of logic could be added as an expansion for `build.py`.
My center of thoughts isn't changed, `build.py` requires simplification, any expansion need to be reasonable.

I agree that more moving parts we are adding the more complexity it will bring over time.

Reply:
afaik, this parameter can be used for dev and prod versions of triton container. A better help text for it would be "Provide any development or released version of project"